code/src/Core/Locations/RemoteTemplatesSource.cs

```diff
@@ -63,11 +72,9 @@ private static void DownloadContent(string sourceUrl, string file)
             }
             catch (Exception ex)
             {
-                string msg = "The templates content can't be downloaded.";
-
-                AppHealth.Current.Exception.TrackAsync(ex, msg).FireAndForget();
-
-                throw;
+                string msg = $"Templates content can't be downloaded right now, we will try it later.";
```

Comment:
That's not how I understand the message. Your comment above contains lots of information that isn't communicated. "we will try it again later" doesn't say when. Nor that there are conditions about trying again.